config/initializers/warden.rb

```diff
@@ -6,19 +6,24 @@
 require 'open_project/authentication/strategies/warden/user_basic_auth'
 require 'open_project/authentication/strategies/warden/session'
 
-strategies = {
-  basic_auth_failure: OpenProject::Authentication::Strategies::Warden::BasicAuthFailure,
-  global_basic_auth:  OpenProject::Authentication::Strategies::Warden::GlobalBasicAuth,
-  user_basic_auth:    OpenProject::Authentication::Strategies::Warden::UserBasicAuth,
-  session:            OpenProject::Authentication::Strategies::Warden::Session
-}
+WS = OpenProject::Authentication::Strategies::Warden
+
+strategies = [
+  [:basic_auth_failure, WS::BasicAuthFailure, 'Basic'],
+  [:global_basic_auth,  WS::GlobalBasicAuth,  'Basic'],
+  [:user_basic_auth,    WS::UserBasicAuth,    'Basic'],
```

Comment:
Okay, so our angular client needs to request session based auth and everything is fine.

Actually when opening the WP list you will get everything: v3, v2 and experimental... what a lovely world this is :)